test/e2e/opentelemetry/instrumentation/opentelemetry.test.ts

```diff
@@ -61,755 +62,762 @@ describe('opentelemetry', () => {
+                        },
+                        kind: 2,
+                        status: { code: 0 },
                       },
-                      kind: 0,
-                      status: { code: 0 },
-                    },
-                    {
-                      name: 'generateMetadata /app/[param]/rsc-fetch/page',
-                      attributes: {
-                        'next.page': '/app/[param]/rsc-fetch/page',
-                        'next.span_name':
-                          'generateMetadata /app/[param]/rsc-fetch/page',
-                        'next.span_type': 'ResolveMetadata.generateMetadata',
+                      {
+                        name: 'generateMetadata /app/[param]/layout',
+                        attributes: {
+                          'next.page': '/app/[param]/layout',
+                          'next.span_name':
+                            'generateMetadata /app/[param]/layout',
+                          'next.span_type': 'ResolveMetadata.generateMetadata',
+                        },
+                        kind: 0,
+                        status: { code: 0 },
                       },
-                      kind: 0,
-                      status: { code: 0 },
-                    },
-                    {
-                      attributes: {
-                        'next.clientComponentLoadCount': isNextDev
-                          ? // In dev, additional client components are being loaded
-                            // due to RSC props being deserialized.
-                            9
-                          : 6,
-                        'next.span_type':
-                          'NextNodeServer.clientComponentLoading',
+                      {
+                        name: 'generateMetadata /app/[param]/rsc-fetch/page',
+                        attributes: {
+                          'next.page': '/app/[param]/rsc-fetch/page',
+                          'next.span_name':
+                            'generateMetadata /app/[param]/rsc-fetch/page',
+                          'next.span_type': 'ResolveMetadata.generateMetadata',
+                        },
+                        kind: 0,
+                        status: { code: 0 },
                       },
-                      kind: 0,
-                      name: 'NextNodeServer.clientComponentLoading',
-                      status: {
-                        code: 0,
+                      {
+                        attributes: {
+                          'next.clientComponentLoadCount': isNextDev
+                            ? // In dev, additional client components are being loaded
+                              // due to RSC props being deserialized.
+                              process.env.TURBOPACK
+                              ? 11
+                              : 9
+                            : process.env.TURBOPACK
+                              ? 8
+                              : 6,
```

Comment:
The `isNextDev` bit wasn't added by me, only the Turbopack-specific cases.
Is it expected that Turbopack leads two 2 additional client components?